packages/jest-diff/src/cli/index.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+import yargs from 'yargs';
+import {readFile} from 'fs';
+
+// eslint-disable-next-line import/default
+import jestDiff from '../';
+import {NO_DIFF_MESSAGE} from '../constants';
+
+/* eslint-disable sort-keys */
+const EXIT_CODES = {
+  // like GNU diff
+  EQUAL: 0,
+  DIFFERENT: 1,
+  INVALID_USAGE: 2,
+
+  // more specific than GNU diff, which exits with 2 in all error cases
+  INTERNAL_ERROR: 10,
+  IO_ERROR: 20,
+  INVALID_JSON: 30,
+};
+/* eslint-enable sort-keys */
+
+const read = (path: string) =>
+  new Promise(resolve =>
+    readFile(path, (readErr, data) => {
```

Comment:
Reading synchronously is faster and there's no other code that would benefit this asynchrony (it's a simple single-purpose CLI tool, not a server :D), please change to `readFileSync` :)